action.yml

```diff
@@ -48,6 +48,18 @@ inputs:
     description: Whether to force the installation of jq
     required: false
     default: "true"
+  sort-by:
+    required: false
+    description: 'Sort by query'
+    default: '.stack_slug'
+  group-by:
+    required: false
+    description: 'Group by query'
+    default: '.stack_slug | split("-") | [.[0], .[2]] | join("-")'
+  nested-matrices-count:
+    required: false
+    description: 'Matrices nested levels count (from 1 to 3)'
+    default: "1"
```

Comment:
Could you provide a more detailed description of what this is? I _think_ this is specifying how many chunks you want? IMO, a more robust solution would be to calculate the entire matrix into N number of 256 item chunks and have an output that tells you the total number of chunks along with a `chunks` output that you can request a certain chunk number, similar to how server-side pagination works in most web frameworks.

Reply:
The case with that setting is caller workflows should support that level of nested matrices with reusable workflows. 
By default, we will use `2` nested levels, `1` is for back compatibility, and `3` is for the future (as we already have this code).